tests/app/helpers.py

```diff
@@ -6,6 +6,9 @@
 from app import create_app
 from app import elasticsearch_client
 
+from ..conftest import (
```

Comment:
It feels like it'd be a little less weird having `conftest.py` import things from the helpers rather than the other way around.

Reply:
I thought so too initially, but then I ended up with circular imports

Reply:
Does fit on one line without the brackets though :)